lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -107,7 +108,7 @@ class RadarChartPainter extends BaseChartPainter<RadarChartData> {
         canvasWrapper.drawCircle(centerOffset, tickRadius, _tickPaint);
         _ticksTextPaint
           ..text = TextSpan(
-            text: tick.toStringAsFixed(1),
+            text: data.getTickTitle?.call(index, tick) ?? tick.toStringAsFixed(1),
```

Comment:
As a convention, we move our default logic into a default implementation of that typedef, and we set the default implementation in the constructor (If the user didn't customize it).
For example please check [here](https://github.com/imaNNeoFighT/fl_chart/blob/2d42fdb0969eb3e3a8eac39637ea96083fe28b94/lib/src/chart/line_chart/line_chart_data.dart#L731).

Reply:
Got it, I'll change it. Thank you!